lib/scrubData.js

```diff
@@ -8,37 +8,58 @@ const scrubData = (data) => {
 
 
 const scrubDataConditions = (data) => {
-    return {
-      status: data.current_observation.weather,
-      location: data.current_observation.display_location.full,
-      tempF: data.current_observation.temp_f,
-      tempC: data.current_observation.temp_c,
-      icon: data.current_observation.icon_url
-    }
+  return {
+    status: data.current_observation.weather,
+    location: data.current_observation.display_location.full,
+    tempF: data.current_observation.temp_f,
+    tempC: data.current_observation.temp_c,
+    icon: data.current_observation.icon_url
   }
+}
 
 const scrubDataHourly = (data) => {
-    return data.hourly_forecast.map((hour) => {
-      return {
-        time: hour.FCTTIME.civil,
-        tempE: hour.temp.english,
-        tempM: hour.temp.metric,
-        condition: hour.condition,
-        icon: hour.icon_url
-      }
-    })
-  }
+  let sevenHours = data.hourly_forecast.filter((val, i) => {
```

Comment:
this is a total nit picky thing, but i try to avoid using `let` for a variable. In this context it doesn't have much of an impact, but the value of `const` is that it is immutable. This can be a good way to avoid weird bugs in code. I tend to use `const` for most variable declarations in my code, unless i am positive that it need to me mutated. this helps create a safeguard by not letting you override a variable type...

obviously, this is not a big deal in this function as it is set once and then mapped and returned. but in larger applications it can help.